test/parallel/test-stringbytes-external.js

```diff
@@ -1,4 +1,5 @@
 'use strict';
+// Flags: --expose_internals
```

Comment:
No, where is `require('binding/…` used in this test? Note that it doesn't seem to call `hasMultiLocalhost`.